deepchecks/nlp/utils/text_properties.py

```diff
@@ -193,6 +193,13 @@ def formality(raw_text: Sequence[str], device: Optional[int] = None) -> List[flo
     classifier = get_transformer_pipeline('formality', model_name, device=device)
     return [x['score'] if x['label'] == 'formal' else 1 - x['score'] for x in classifier(raw_text)]
 
+def lexical_density(raw_text: Sequence[str]) -> List[str]:
+    """Return list of floats of lexical density."""
```

Comment:
add a few more word and also maybe link to website explaining what is lexical_density

Reply:
Done